h/api/search/test/core_test.py

```diff
@@ -77,20 +91,20 @@ def test_search_queries_for_replies(query, models):
     assert builder.append_matcher.call_args_list[-1] == mock.call(
         query.RepliesMatcher.return_value)
 
-    # It should call search_raw() a second time with the body from the
+    # It should call search() a second time with the body from the
     # query builder.
-    assert models.Annotation.search_raw.call_count == 2
-    last_call = models.Annotation.search_raw.call_args_list[-1]
-    first_arg = last_call[0][0]
-    assert first_arg == builder.build.return_value
+    assert request.es.conn.search.call_count == 2
+    _, last_call_kwargs = request.es.conn.search.call_args_list[-1]
+    assert last_call_kwargs['body'] == builder.build.return_value
 
 
 @search_fixtures
 def test_search_returns_replies(models):
-    """It should return an Annotation for each reply from search_raw()."""
-    models.Annotation.search_raw.side_effect = [
-        # The first time search_raw() is called it returns the result of
-        # querying for the top-level annotations only.
+    """It should return an Annotation for each reply from search()."""
```

Comment:
Ah, no, wait. This is still correct. The switch to returning dicts happens in #2866.